dev-libs/openssl/files/gentoo.config-1.0.2

```diff
@@ -162,6 +163,9 @@ mingw*)
 	# special case ... no xxx-yyy style name
 	echo ${system}
 	;;
+hurd)
+	machine=x86
```

Comment:
Surely this cannot be right. Why x86?

Reply:
Because configure scripts expect 'hurd-x86'

Reply:
True, unconditional `machine=x86` looks ugly. Fixed.